network/wsNetwork.go

```diff
@@ -581,9 +580,9 @@ func (wn *WebsocketNetwork) GetPeers(options ...PeerOption) []Peer {
 }
 
 func (wn *WebsocketNetwork) setup() {
-	var preferredResolver *dnssec.Resolver
+	var preferredResolver dnssec.ResolverIf
 	if wn.config.DNSSecurityRelayAddrEnforced() {
-		preferredResolver = &dnssec.DefaultResolver
+		preferredResolver = dnssec.MakeDefaultDnssecResolver(wn.config.FallbackDNSResolverAddress)
```

Comment:
`MakeDefaultDnssecResolver` has stuff from `/etc/resolv.conf` and `FallbackDNSResolverAddress` that might fail to handle DNSSEC requests. Maybe it makes sense to add track failed server and do not query them again to speedup things